subprojects/packagefiles/speexdsp/meson.build

```diff
@@ -0,0 +1,136 @@
+project('speexdsp',
+        'c',
+        version : '1.2.0',
+        license : 'BSD-3-Clause',
+        default_options : [
+          'c_std=c99',
+          'buildtype=release',
+          'default_library=both',
+          'b_lto=true',
+          'warning_level=0'])
+
+summary('Build type',
+        get_option('buildtype'),
+        section : 'Build Summary')
+
+summary('Link time optimization',
+        get_option('b_lto'),
+        section : 'Build Summary')
+
+summary('Library type',
+        get_option('default_library'),
+        section : 'Build Summary')
+
+summary('Install prefix',
+        get_option('prefix'),
+        section : 'Build Summary')
```

Comment:
From the docs for summary:

> Instead of calling summary as summary(key, value), it is also possible to directly pass a dictionary to the [summary()](https://mesonbuild.com/Reference-manual_functions.html#summary) function, as seen in the example below.

You could do e.g.

```meson
summary({
    'Build type': get_option('buildtype'),
    'Link time optimization': get_option('b_lto'),
    'Library type': get_option('default_library'),
    'Install prefix': get_option('prefix'),
}, section: 'Build Summary')
```